bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -227,7 +227,7 @@ const BaseContainer = withTracker(() => {
     if (meetingEnded) Session.set('codeError', '410');
   }
 
-  const approved = Users.findOne({ userId: Auth.userID, approved: true, guest: true });
+  const approved = !!Users.findOne({ userId: Auth.userID, approved: true, guest: true });
```

Comment:
Could we do that for `ejected` too please? Also, now that both will be Booleans, consider renaming to `wasApproved` and `wasEjected`